setup.py

```diff
@@ -18,7 +18,7 @@
     author="Vincent Teo",
```

Comment:
Do you want to add your name here too (comma separated)

Reply:
Done.